components/search/MoreFiltersDropdownFacet.vue

```diff
@@ -31,61 +31,59 @@
           :index="index + limitTo"
         />
       </div>
+      <button
+        v-if="fields.length > limitTo"
+        type="button"
+        class="btn btn-link btn-toggle"
+        :class="{ 'is-active': isActive }"
+        :data-qa="(isActive ? $t(`facets.button.showLess`, { label: $tc(`facets.${name}.name`, 2) }) + ' button' : $t(`facets.button.showAll`, { label: $tc(`facets.${name}.name`, 2) }) + ' button')"
+        @click.prevent="isActive = !isActive"
+      >
+        {{ isActive ? $t(`facets.button.showLess`, { label: $tc(`facets.${name}.name`, 2) }) : $t(`facets.button.showAll`, { label: $tc(`facets.${name}.name`, 2) }) }}
+      </button>
     </b-form-checkbox-group>
-    <button
-      v-if="fields.length > limitTo"
-      type="button"
-      class="btn btn-link btn-toggle"
-      :class="{ 'is-active': isActive }"
-      :data-qa="(isActive ? $t(`facets.button.showLess`, { label: $tc(`facets.${name}.name`, 2) }) + ' button' : $t(`facets.button.showAll`, { label: $tc(`facets.${name}.name`, 2) }) + ' button')"
-      @click.prevent="isActive = !isActive"
-    >
-      {{ showMoreOrLess }}
-    </button>
-  </b-dropdown-form>
+  </b-form-group>
 </template>
 
 <script>
   import MoreFiltersDropdownFacetOption from './MoreFiltersDropdownFacetOption';
+
   export default {
     components: {
       MoreFiltersDropdownFacetOption
     },
+
     props: {
       fields: {
         type: Array,
         required: true
       },
+
       name: {
         type: String,
         required: true
       },
+
       selected: {
         type: Array,
         default: () => []
       }
     },
+
     data() {
       return {
         selectedOptions: this.selected,
         isActive: false,
         limitTo: 9
       };
     },
-    computed: {
-      showMoreOrLess() {
-        if (this.isActive) {
-          return this.$t('facets.button.showLess', { label: this.$tc(`facets.${this.name}.name`, 2).toLowerCase() });
-        } else {
-          return this.$t('facets.button.showAll', { label: this.$tc(`facets.${this.name}.name`, 2).toLowerCase() });
```

Comment:
these .toLowerCase statements need to come across on the refactor.

Reply:
Indeed. That's been restored.